src/components/Popups/TextArea/TextArea.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import classnames from 'classnames';
+
+export type Props = {
+    className?: string;
+    defaultValue?: string;
+    onChange: (event: React.ChangeEvent<HTMLTextAreaElement>) => void;
+    onClick: (event: React.SyntheticEvent) => void;
+    textareaRef?: React.Ref<HTMLTextAreaElement>;
+};
+
+class TextArea extends React.Component<Props> {
```

Comment:
This can be a function component.

Reply:
> This can be a function component.

Yeah, but in the next ticket, we need state to store mentions

Reply:
These components should almost always rely on the redux store for their state, even if it's ephemeral. The root node can be removed/recreated at any time and any local state will be lost.